fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/domain/PreferTagTest.java

```diff
@@ -32,4 +35,15 @@ public void testEmpty() {
         assertEquals("", preferTag.getValue());
         assertTrue(preferTag.getParams().isEmpty());
     }
+
```

Comment:
Please add the following tests:
assertFalse(preferTag1.equals(null));
assertFalse(preferTag1.equals("some string"));